docs/source/tutorials/psij.ipynb

```diff
@@ -37,478 +26,614 @@
+      "  Cloning https://github.com/ExaWorks/psij-python.git to /tmp/pip-req-build-pdd8uqa7\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "  Running command git clone --filter=blob:none --quiet https://github.com/ExaWorks/psij-python.git /tmp/pip-req-build-pdd8uqa7\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "  Resolved https://github.com/ExaWorks/psij-python.git to commit 1d9bdd8edf90f8dbfc5e3646a2cb2f2f2626703c\n",
+      "  Preparing metadata (setup.py): started\n",
+      "  Preparing metadata (setup.py): finished with status 'done'\n",
+      "Collecting filelock\n",
+      "  Downloading filelock-3.9.0-py3-none-any.whl (9.7 kB)\n",
+      "Requirement already satisfied: psutil in /opt/conda/lib/python3.9/site-packages (from psij-python==0.1.0.post2) (5.9.2)\n",
+      "Collecting pystache>=0.6.0\n",
+      "  Downloading pystache-0.6.0.tar.gz (78 kB)\n",
+      "     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 78.2/78.2 kB 1.8 MB/s eta 0:00:00\n",
+      "  Installing build dependencies: started\n",
+      "  Installing build dependencies: finished with status 'done'\n",
+      "  Getting requirements to build wheel: started\n",
+      "  Getting requirements to build wheel: finished with status 'done'\n",
+      "  Preparing metadata (pyproject.toml): started\n",
+      "  Preparing metadata (pyproject.toml): finished with status 'done'\n",
+      "Requirement already satisfied: typeguard in /opt/conda/lib/python3.9/site-packages (from psij-python==0.1.0.post2) (2.13.3)\n",
+      "Building wheels for collected packages: psij-python, pystache\n",
+      "  Building wheel for psij-python (setup.py): started\n",
+      "  Building wheel for psij-python (setup.py): finished with status 'done'\n",
+      "  Created wheel for psij-python: filename=psij_python-0.1.0.post2-py3-none-any.whl size=76556 sha256=b53305089556030549b27dce7fbb9144dd6da1bdc89ad2a0feadcf45c04a615e\n",
+      "  Stored in directory: /tmp/pip-ephem-wheel-cache-rk8_ikuy/wheels/37/37/2d/dbe40f9e59bec39469bdf5e7237b873a61e7d41daaa5f283a4\n",
+      "  Building wheel for pystache (pyproject.toml): started\n",
+      "  Building wheel for pystache (pyproject.toml): finished with status 'done'\n",
+      "  Created wheel for pystache: filename=pystache-0.6.0-py3-none-any.whl size=83618 sha256=cbf832367bbb06ad27afd1809549f2b75779bf94851e045e5ea1e98b54305f56\n",
+      "  Stored in directory: /home/jovyan/.cache/pip/wheels/56/e7/49/8561f499b433df173f5ffe04a490876fadc8f51ab62e3a5003\n",
+      "Successfully built psij-python pystache\n",
+      "Installing collected packages: pystache, filelock, psij-python\n",
+      "Successfully installed filelock-3.9.0 psij-python-0.1.0.post2 pystache-0.6.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "pip install git+https://github.com/ExaWorks/psij-python.git"
```

Comment:
maybe keep original approach to eliminate installation logs?
```
%%capture capt
%pip install ...
```
and with the next cell to show what was installed
```
pip show psij-python
```

Reply:
Indeed that is cleaner. I changed to a variation of that. Thank you.